dhlab/legacy/graph_networkx_louvain.py

```diff
@@ -500,3 +500,8 @@ def reduce_MxM_graph(G, words, factor=0.01):
             edges.append((w1, w2, new_weight))
     Gm.add_weighted_edges_from(edges)
     return Gm
+
+def return_networkx_digraph(edgelist):
```

Comment:
Seeing as `return` is a python keyword, I'd prefer the function signature to use another word, e.g. `create` or `construct`:

```suggestion
def create_networkx_digraph(edgelist: list):
```